docs/user/exams/instructors_guide.rst

```diff
@@ -0,0 +1,192 @@
+
+    - **Hint:** You can change the individual ``working time`` of students from here. 
+
+- To generate the student exams, you must click on ``Generate individual exams``. This will automatically create a student exam for every registered user. The number of exercises will be determined by the flag set in the exam configuration, see `Create and Configure Exam`_. The exercises itself will be randomly selected from the available exercise variants per exercise group, see `Exercise Groups`_.
+- The ``Generate individual exams`` button will be locked once the exam becomes visible to the students. You cannot perform changes to the student exams once the :ref:`exam conduction <exam_conduction>` has started. 
+- If you have added more students recently, you can choose to ``Generate missing individual exams``. 
+- ``Prepare exercise start`` creates a participation for each exercise for every registered user, based on their assigned exercises. It also creates the individual repositories and build plans for programming exercises. This action can take a while if there are many registered students due to the communication between the VC and CI server. 
+
+    - **Warning:** ``Prepare exercise start`` must be executed before the :ref:`exam conduction <exam_conduction>` begins. 
+
+- On the *Student Exams* page, you can also maintain the repositories of the student exams. You can choose to ``lock the repositories`` and ``unlock`` them.
+- Additionally, once the :ref:`exam conduction <exam_conduction>` is over, you can click on ``Evaluate quizzes``. This action will evaluate all student exam submissions for all quiz exercises and assign an automatic result. 
+
+   - **Hint:** If you do not press this button, the students quiz exercise will not be graded.
+
+.. figure:: instructor/student_exams.png
+   :alt: Student Exam Page
+   :align: center
+
+   Student Exam Page
+
+Assess Student Exams and Complaints
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- Once the :ref:`exam conduction <exam_conduction>` is over and the latest individual ``working time`` has passed, your team can begin the :ref:`assessment <exam_assessment>` process. 
+- This is done through the *Tutor Exam Dashboard*. 
+
+    - **Hint:** If the :ref:`exam conduction <exam_conduction>` is not over, you will not be able to access this page.
+
+- The :ref:`assessment <exam_assessment>` process is anonymised. All student information will not be displayed to the assessors.
+- The ´*Tutor Exam dashboard* provides an overview over the current assessment progress per exercise. For each exercise, you can view how many submissions have already been assessed and how many are still left. The status of the student complaints is also displayed here. 
+
+    - **Hint:** To check for plagiarism, you must navigate to the individual exercise. This can be done by navigating to:
```

Comment:
This needs more explanation. Please add a screenshot with an example for modeling exercises as well